hack/manifests/csi/vsphere/02-example-namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: windows-webserver
+  labels: 
+    security.openshift.io/scc.podSecurityLabelSync: "false"
+    pod-security.kubernetes.io/enforce: privileged
```

Comment:
Are these labels required?

Reply:
Yes, they are required so the pod has the appropriate privileges/sccs

Reply:
Consider stating that inline with a comment.